tutor/resources/styles/components/top-nav-bar.less

```diff
@@ -21,49 +52,45 @@
     }
   }
 
-  background: @tutor-white;
-  z-index: @zindex-navbar-fixed;
-  position: fixed;
-  top: 0;
-  right: 0;
-  left: 0;
-  z-index: @zindex-navbar-fixed;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  height: @tutor-navbar-height;
-
-
-  padding-right: @navbar-padding-horizontal;
-  padding-left:  @navbar-padding-horizontal;
-
   a {
-    color: #777;
+    color: inherit;
     text-decoration: none;
+
+    &:hover,
+    &[aria-expanded="true"] {
+      color: #333;
+    }
   }
 
   > * {
     display: flex;
-    align-items: center;
   }
+
   .ui-brand-logo {
-    margin-top: 3px; // the text on the logo is a bit vertically mis-aligned
     display: inline-block;
-    height: 41px;
-    width: 265px;
+    margin-top: 12px;
+    height: 34px;
+    width: 200px;
     background-repeat: no-repeat;
```

Comment:
Don't have a preference on the method used, so long as the logo and menu items are aligned to the same baseline.